transports/rmi/src/test/java/org/mule/transport/AbstractFunctionalTestCase.java

```diff
@@ -46,13 +52,28 @@ public void testCase() throws Exception
 
         // send String
         message = client.send("vm://testin", "test matching component first time", null);
-        assertNotNull(message);
-        assertEquals(message.getPayload(), "emit tsrif tnenopmoc gnihctam tset");
+        if(variant.equals(SERVICE))
```

Comment:
Yes It is. 
As in 3.x the default behavior changes, I had to modify some tests.